libraries/streaming-e2e-tests/build/create-react-app.yml

```
@@ -0,0 +1,37 @@
+# Create a React app that uses ReactWebChat
+
+steps:
+  - powershell: npx create-react-app react-app
+    displayName: 'create-react-app'
+    workingDirectory: $(RootReactPiecesDir)
+
+  - powershell: npm install botframework-webchat
```

```suggestion
  - powershell: npm install botframework-webchat@4.9.2
```

Please lock down on version of Web Chat. This is because the DOM tree of Web Chat may change from time to time and it will affect your test code.

(Or file an issue to ask Web Chat to publish a page object library)